packages/common/src/TitleBar.ts

```diff
@@ -34,6 +36,14 @@ export class Button extends HTMLWidget {
         element
             .classed("disabled", !this.enabled())
             .attr("title", this.tooltip())
+            .style("display", "inline-block")
+            .style("height", this.fixedHeight() + "px")
+            .style("line-height", this.fixedHeight() + "px")
+            .style("min-width", this.fixedHeight() + "px")
```

Comment:
Is this deliberate?

Reply:
Yes, it ensures the buttons are square (or wider if the text content needs it to be)

Reply:
Ok (just checking)